parallel_cluster/README.md

```diff
@@ -93,6 +93,20 @@ from the `ec2-user`, this is handy for debugging purposes:
 `ssh local-ip-of-running-compute node`.  
 Use sinfo to see the current list of running compute nodes.
 
+### Staging input and reference data
+You will likely need to download your input data and software.  
+Ensure that inputs are accessible to all nodes by placing it in the `${SHARED_DIR}` folder.  
+This is:
+  * `/efs` if using the efs configuration on `tothill` or `umccr_dev`
+  * `/fsx` if using the fsx configuration on `umccr_dev_fsx`
+  
+By default you will have read-only access to the s3 buckets.  
+You may use `aws s3 sync s3://<bucket_path> "${SHARED_DIR}/local_path` to download data
+into the shared file system.  
+
+> Compute nodes have a much higher bandwidth than the master nodes.    
```

Comment:
I would put that bigger/bold for users to notice... along with an sbatch script example for "data copy purposes".